channel.py

```diff
@@ -151,7 +151,7 @@ def name(self) -> str:
 
     @property
     def url(self) -> URLType:
-        return URLType(f"{BASE_URL}/{self._login}")
+        return URLType(f"{BASE_URL_MOBILE}/{self._login}")
```

Comment:
Not entirely sure what to do here, honestly. Dropping the `ANDROID_APP` and `SMARTBOX` clients could be a solution for this, but I'd rather leave them in. It'd probably be best to just bind the `ClientInfo` instance to the `Twitch` instance (with a new `client_info` attribute), and then use `self._twitch.client_info.CLIENT_URL` here. A new constant isn't really needed. The new binding would remove the need to import `BASE_URL` into this file entirely.